### PR TITLE
Adds nausea

### PR DESCRIPTION
**Pull request by monster860**
[https://gfycat.com/YellowishEnergeticChimpanzee](https://gfycat.com/YellowishEnergeticChimpanzee)
a
It's a more primitive version of the nausea effect in minecraft. (BYOND is only capable of skewing stretching rotating and translating.)

I'll embed it in the post later.

:cl: monster860
rscadd: Adds nausea
/:cl:

### CODE REVIEW

**Comment by QQCucumber** [comment]
The effect might be too good because I can see this giving me nausea irl

**Comment by Fox-McCloud** [comment]
It's been over 10 years since I last vomited, but what I remember of it, my head most certainly did not spin prior to the actual event. 

This makes sense for something like drunkennes (where it would be perfect iMO), but just vomitting in general? Don't think it's a good idea, and that's way too much wobble to begin with---for reasons @QQCucumber has stated.

**Comment by CrAzYPiLoT-SS13** [comment]
I don't really like the effect. Lessen the skew a little bit just so it
doesn't look like a 3d picture being moved around, then it might be good.

**Comment by pinatacolada** [comment]
if you could scale it like the drunken shaking thing it'd be pretty neat for being drunk, but it needs to be toned down a bit

**Comment by monster860** [comment]
The intensity in the pic is .25

The intensity actually in the pr is .17

**Comment by Aurorablade** [comment]
For the courtsey of other player and players with motion sickness issues. No. Please no.

**Comment by Keilden** [comment]
I like it alot, add some filters to it to make it even more sick.

**Comment by VampyrBytes** [comment]
I think it's a very effective effect... but just the thought of trying to play like that makes me feel nauseous

**Comment by Regen1** [comment]
So make a gif with intensity 17, not 25

**Comment by monster860** [comment]
I was running pate when I opened the pr

File: code/modules/mob/mob_defines.dm [hunk 1]
```diff
@@ -93,6 +93,7 @@
 	var/flying = 0
 	var/charges = 0.0
 	var/nutrition = 400.0//Carbon
+	var/nausea//Carbon
```
Comment (tigercat2000) [inline code/modules/mob/mob_defines.dm]: let's not add any more awful `//Carbon` comments
anything can use this

**Comment by Vivalas** [comment]
I like Bay's nausea effect better IMO, but this is good too.

**Comment by CrAzYPiLoT-SS13** [comment]
By the way, not really sure the HUD should be affected.

**Comment by monster860** [comment]
@CrAzYPiLoT-SS13 That would require me to move the hud to a different `plane`, and BYOND has yet to implement FLOAT_PLANE

**Comment by KasparoVy** [comment]
Pretty wicked effect. @Vivalas Mind posting what Bay's effect looks like, for posterity?

**Comment by monster860** [comment]
@CrAzYPiLoT-SS13 Also, it adds to the fun of trying to use your items. The click handles move too when the screen wobbles, so you have to click on where it shows on the screen.

**Comment by tigercat2000** [comment]
meehh
I think this looks kinda cool
but it's going to make some people motion sick as hell
and it's annoying as fuck to have caused when you would otherwise just puke on the floor with no stun `fakevomit()`

**Comment by DavidRossii** [comment]
The UI shaking around with it just looks dumb.

**Comment by taukausanake** [comment]
@DavidRossii I think it's like that because even your senses are confused and so trying to do anything will be tough unless you focus. I do agree though, it looks silly moving around

**Comment by Iamgoofball** [comment]
HAHAHAHAHAHA

YES

**Comment by Iamgoofball** [comment]
gotta GET A GRIP

**Comment by Spacemanspark** [comment]
I like it.

**Comment by KasparoVy** [comment]
Something to keep in mind is that the more proficient you are at the usage of hotkeys, the less this will effect you.

**Comment by TheDZD** [comment]
@monster860 What does this look like moving around? I can imagine it'll probably look a lot more disorienting when combined with movement.

**Comment by monster860** [comment]
I'll probably need the server updated to the latest version of BYOND for UI to work properly.

**Comment by Fullofskittles** [comment]
As someone who frequently does suffer from motion sickness/headaches because of effects like this in video games...please no. Even with slower speeds this'll be overwhelming, more so if you consider that people will be moving around and you'll probably be moving around too.

What all is this supposed to apply to? When would you encounter it in game and could it be easily avoided and 'cured'? I really don't want to have to worry about being forcefed/injected with/smoke bomb'd with things that cause this effect.

**Comment by Fox-McCloud** [comment]
>What all is this supposed to apply to

Just about any time you vomit, this will occur.

**Comment by CrAzYPiLoT-SS13** [comment]
Oh God, I didn't know this applied every single time you vomited. Walk past
a corpse? Struggle to get away from it cause of the effect. Ate something
bad? Fall unconscious because of how long it took you to go to medbay.
Please no. Make it apply in special occasions only. For example, when you
get sleepy penned, this might be a nice effect paired with a gradual screen
darkening.

**Comment by Vivalas** [comment]
image

@KasparoVy

Very light swaying that increases over time.

**Comment by Vivalas** [comment]
We actually already have this effect... didn't know that...


This PR seems redundant then.. :-1: , I like the current nausea better.

**Comment by Fox-McCloud** [comment]
Considering how negative the reaction this has received, the real possibility of inducing motion sickness, and the fact that it's already redundant with some effects of drunkenness, going to close this.

**Pull request closed without merging**